docs/DeveloperGuide.md

```diff
@@ -238,6 +238,27 @@ _{more aspects and alternatives to be added}_
 
 _{Explain here how the data archiving feature will be implemented}_
 
+### Get inpatients and outpatients feature (`get /inp` & `get /outp`)
+
```

Comment:
In the coming week perhaps we could have one overarching `Get` feature, and place all subcommands under it? @nehcuy